# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/daniel/gkrellm-plugins/gkrellmms_hotfix.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./gkrellmms.c.orig	2004-09-07 13:42:50.228933752 +0200
+++ ./gkrellmms.c	2004-09-07 13:45:13.870096976 +0200
@@ -672,7 +672,7 @@
 		y = y * gkrellm_get_theme_scale();
 #endif
 		if (text)
-			y -= decal->y_baseline;
+			y -= decal->y_ink;
 		gkrellm_move_decal(control_panel, decal, x, y);
 		}
 	}
